rosdep/python.yaml

```diff
@@ -893,7 +897,7 @@ python-mako:
   ubuntu: [python-mako]
 python-mapnik:
   fedora: [mapnik-python]
-  ubuntu: [python-mapnik2]
+  ubuntu: [python-mapnik]
```

Comment:
Does this need to be separated out into the different supported Ubuntu versions? Or is `python-mapnik` actually used on all versions, not just Xenial?